bin/import_xml2obd.py

```diff
@@ -41,6 +44,20 @@
 canada_resource_format = None
 
 
+def md5str(fname):
+    return hashlib.md5(open(fname, 'rb').read()).hexdigest().strip()
+
+def md5_file(fname):
+    return hashlib.md5(open(fname, 'rb').read()).digest()
```

Comment:
*if* you need to compute the hash yourself it's better to do it in chunks of a few MB at a time instead of by loading the whole file into memory. Even better is not needing to compute hash values.